Dockerfile

```diff
@@ -0,0 +1,29 @@
+ARG NODE_VERSION=20
+ARG OS_VERSION=bullseye-slim
+# Content of this file derived from 
+# https://github.com/nodejs/docker-node/blob/main/docs/BestPractices.md
+FROM node:${NODE_VERSION}-${OS_VERSION}
+
+# Build/Run time args
+ARG APP_DIR=mDL-Testing
+ARG NODE_USER=node
+ARG NODE_HOME=/home/${NODE_USER}
+
+# TODO: this currently prevent npm run build to work.
+#ENV NODE_ENV=production
```

Comment:
What error do you get when you uncomment this?

When you specify `NODE_ENV=production`, `npm install` will not install `devDependencies` specified in the package.json.

Reply:
@NimeshNB can we address this problem by shifting `devDependencies` to regular dependencies, thus enabling `NODE_ENV=production` ?

Reply:
Yes sure i'll try to move ```devDependencies``` to regular ```dependencies```